apps/script/build-color-js.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+// Script to generate color.js from color.scss
+var fs = require('fs');
+var path = require('path');
+var readline = require('readline');
+
+var colorScssPath = path.resolve('../shared/css/color.scss');
+var colorJsPath = path.resolve('./src/color.js');
+
+// Regular expression to capture a color definition from SCSS
+var colorRe = /\$(\w+)\s*:\s*([^;]+);/;
```

Comment:
This regex captures any scss variable definition, so we're sort of creating a new constraint on color.scss that says we should keep it relatively simple and only define colors there.  I think that was always implied, but this script formalizes it somewhat.